### PR TITLE
351 mobile layout

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: www/modules/calendar/less/_calendar.base.less [hunk 1]
```diff
@@ -91,7 +91,7 @@
 
     .fc-axis {
       color: @calendar-secondary-font-color;
-      text-align: left;
+      text-align: left;#1E3141
```
Comment (ghost) [inline www/modules/calendar/less/_calendar.base.less]: Seems it is a typo

File: www/modules/core/less/core.less [hunk 2]
```diff
@@ -17,6 +17,7 @@
 .main {
   &-view-container {
     height: 100%;
+    overflow: scroll;
```
Comment (ghost) [inline www/modules/core/less/core.less]: Could we use `auto` instead `scroll`?

File: www/modules/users/views/authentication/signin.client.view.html [hunk 3]
```diff
@@ -51,14 +51,15 @@ <h1 class="app-logo__description">Make It Happen</h1>
                     type="submit"> Sign in
             </button>
             <a class="login-view__btn login-view__btn--blue"
-               ui-sref="auth.signup">Create an account</a>
+               ui-sref="auth.signup">Create account</a>
             <div class="auth-form-wrapper login-form-container--singin">
                 <a class="auth-form-wrapper__link--forgot" ui-sref="auth.forgot">
-                    Forgot your password?</a>
+                    Forgot password?</a>
```
Comment (ghost) [inline www/modules/users/views/authentication/signin.client.view.html]: Please, add `data- `prefix to` ui-sref` on links

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Do we really need  `www/modules/users/img/backgrounds/bg--1.png` image?
We already have the `same mih-todo-app/www/modules/users/img/backgrounds/bg-1.jpg`. 
Coould we just change background position?

File: www/modules/calendar/less/_calendar.base.less (hunk 1 above)
Comment (ghost) [reply www/modules/calendar/less/_calendar.base.less]: fixed

File: www/modules/users/views/authentication/signin.client.view.html (hunk 3 above)
Comment (ghost) [reply www/modules/users/views/authentication/signin.client.view.html]: fixed

File: www/modules/core/less/core.less (hunk 2 above)
Comment (ghost) [reply www/modules/core/less/core.less]: Yes> we can. Fixed

**Comment by ghost** [comment]
We have 404 according to www/modules/users/img/backgrounds/bg--1.png image/ File was deleleted and didnt displayed in login veiw

File: www/modules/sidebar/controllers/sidebar.client.controller.es6 [hunk 4]
```diff
@@ -1,7 +1,15 @@
...
-  constructor(Authentication) {
-    this.user = Authentication.user;
+  /*@ngInject*/
+  constructor(Authentication, $state) {
+    Object.assign(this, {Authentication, $state});
+    this.user = this.Authentication.user;
+  }
+  singOut (){
+    this.state.go('auth.signin');
+    this.user = undefined;
```
Comment (ghost) [inline www/modules/sidebar/controllers/sidebar.client.controller.es6]: Why we assign undefined and not null?
I think null is more suitable in this case.

File: www/modules/users/controllers/settings.client.controller.es6 [hunk 5]
```diff
@@ -1,37 +1,33 @@
...
-    this.Notification = $injector.get('Notification');
+    if (!this.Authentication.user) {
+      this.$location.path('/');
+    }
 
-    /*fields*/
+    this.user        = this.Authentication.user;
+    this.imageURL    = this.user.profileImageURL;
     this.workingDays = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun'];
     this.reminders   = [5, 10, 15, 20, 25, 30];
```
Comment (ghost) [inline www/modules/users/controllers/settings.client.controller.es6]: We don't have reminder functionality. Could you please remove redundant dead code?

File: www/less/_app.variables.less [hunk 6]
```diff
@@ -122,6 +122,7 @@
 @login-input-font-color: @color-bali-hai-blue-s25;
 @login-font-color: @light-text;
 @login-input-border-custom-color: @light-text;
+@login-input-border-custom-mobile-color: #507299;
```
Comment (ghost) [inline www/less/_app.variables.less]: Could you please create base variable for this color?
According to www.color-blindness.com

**Comment by AinuraAliieva** [comment]
"...We have 404 according to www/modules/users/img/backgrounds/bg--1.png image/ File was deleleted and didnt displayed in login veiw..."

Yes, but we can use existing image for that.

File: www/modules/calendar/less/_calendar.base.less [hunk 7]
```diff
@@ -57,6 +57,7 @@
   .fc-widget-header {
     border: 0;
     background: @calendar-day-header-bg-color;
+    width: 50px!important;
```
Comment (8bitmousepunk) [inline www/modules/calendar/less/_calendar.base.less:60]: Can we avoind "!important" ?

File: www/modules/sidebar/views/sidebar.client.view.html [hunk 8]
```diff
@@ -1,4 +1,5 @@
-<div class="sidebar">
+<div class="sidebar"
+    data-ng-controller="SidebarController as sidebarCtrl">
```
Comment (8bitmousepunk) [inline www/modules/sidebar/views/sidebar.client.view.html]: We defined controller for sidebar in core routes, so this is redundant.

File: www/modules/sidebar/controllers/sidebar.client.controller.es6 (hunk 4 above)
Comment (ghost) [reply www/modules/sidebar/controllers/sidebar.client.controller.es6]: Fixed comments after review

File: www/modules/users/controllers/settings.client.controller.es6 (hunk 5 above)
Comment (ghost) [reply www/modules/users/controllers/settings.client.controller.es6]: Fixed

File: www/modules/calendar/less/_calendar.base.less (hunk 7 above)
Comment (ghost) [reply www/modules/calendar/less/_calendar.base.less:60]: I use it to override inline styles from library

File: www/modules/sidebar/views/sidebar.client.view.html (hunk 8 above)
Comment (ghost) [reply www/modules/sidebar/views/sidebar.client.view.html]: Fixed

**Pull request merged**